api/openapi/specs/api.yaml

```diff
@@ -514,20 +531,35 @@ components:
           zoneIngresses:
             online: 1
             total: 1
-          dataplanes:
-            online: 23
-            offline: 10
-            partiallyDegraded: 17
-            total: 50
-          policies:
-            total: 100
-          meshes:
-            total: 3
-          resources:
-            MeshTrafficPermission:
-              total: 42
-            MeshService:
-              total: 84
+        dataplanes:
+          standard:
+            total: 4
+            online: 2
+            offline: 1
+            partiallyDegraded: 1
+          online: 23
+          offline: 10
+          partiallyDegraded: 17
+          total: 51
```

Comment:
Does `dataplanes` have those properties?
https://github.com/kumahq/kuma/blob/7e4a0efa1567762a63aef57f598eabcecba4a5c0/api/openapi/specs/api.yaml#L349-L361

Reply:
Yes, otherwise the linter would've not complained.

Reply:
Good point, it was a leftover.